src/main/g8/project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 logLevel := Level.Warn
 
-addSbtPlugin("org.scalameta" % "sbt-scalafmt"  % "2.3.2")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt"  % "2.4.0")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```

Comment:
If we have scoverage, I don't see why not to include coverallls as former without latter is pretty much useless.

Reply:
Done.